.github/workflows/main.yml

```diff
@@ -67,6 +73,7 @@ jobs:
       run: |
         invoke lint
     - name: Coveralls report
+      continue-on-error: true
```

Comment:
let's keep this

Reply:
all `continue-on-error` will be removed before merge 👍

Reply:
thanks